src/Microsoft.Health.Dicom.Core/Microsoft.Health.Dicom.Core.csproj

```diff
@@ -32,6 +32,7 @@
     <PackageReference Include="Microsoft.Health.Operations" />
     <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" />
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="SixLabors.ImageSharp" PrivateAssets="All" />
```

Comment:
This isn't actually an indirect dependency, is it?

Reply:
thanks for the catch